fe/src/assets/css/tailwind.css

```diff
@@ -29,3 +29,62 @@ body {
   white-space: nowrap;
   width: 100%;
 }
+/* Paginator */
+.paginator-arrow-heads {
```

Comment:
No need to create a custom class here in tailwind.css.
Just apply these classes directly to your paginator component elements.